src/raster/r.damflood/main.c

```diff
@@ -163,29 +163,29 @@ int main(int argc, char *argv[]){
   } startpt;
 
   startpt *a_start,*tmp_start;*/
-  struct Cell_head cellhd; /* it stores region information and header information of rasters */
+  struct Cell_head cellhd; /* Stores region information and header information of rasters */
   struct Cell_head window;
-  //struct History history;  /* holds meta-data */
+  //struct History history;  /* Holds metadata */
 
   /* input-output raster file descriptor */
   int infd_ELEV,infd_LAKE,infd_DAMBREAK, infd_MANNING, infd_U, infd_V;
   int outfd_H,outfd_VEL,outfd_VEL_DIR,outfd_HMAX,outfd_T_HMAX,outfd_I_HMAX;
   int outfd_VMAX,outfd_T_VMAX,outfd_I_VMAX,outfd_DIR_VMAX,outfd_IMAX,outfd_T_IMAX,outfd_WAVEFRONT;
-  //float g=9.81; define iniziale!
+  //float g=9.81; Initial definition!
```

Comment:
Was set with `#define g 9.81` at line 51. I'm sure I'm wrong, but it's superfluous here in my opinion.